### PR TITLE
Add MySQL database support

### PR DESCRIPTION
**Pull request by liangfb**
Recently, I needed to use this excellent tool to import the Ethereum data to a MySQL-compatible database, but it doesn't support it yet, so I changed some code and contributed.

### CODE REVIEW

File: blockchainetl/streaming/mysql_utils.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from sqlalchemy.dialects.postgresql import insert
```
Comment (ezioruan) [inline blockchainetl/streaming/mysql_utils.py]: the insert is from PostgreSQL ?

Comment (liangfb) [reply blockchainetl/streaming/mysql_utils.py]: Oh, it is my mistake, it should be from MySQL, I will re-create a PR later.

Comment (ezioruan) [reply blockchainetl/streaming/mysql_utils.py]: you don't need to re-create PR. just push new commits

File: ethereumetl/streaming/mysql_tables.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
+
+from sqlalchemy import Table, Column, Integer, BigInteger, Boolean, String, Numeric, \
+    MetaData, PrimaryKeyConstraint, VARCHAR, DATETIME, MEDIUMTEXT
+
+metadata = MetaData()
+
+# SQL schema is here https://github.com/blockchain-etl/ethereum-etl-postgres/tree/master/schema
```
Comment (ezioruan) [inline ethereumetl/streaming/mysql_tables.py:7]: table schema looks good, but index is missing here

**Pull request closed without merging**